google/resource_google_project.go

```diff
@@ -438,9 +446,13 @@ func updateProjectBillingAccount(d *schema.ResourceData, config *Config) error {
 		return fmt.Errorf("Error setting billing account %q for project %q: %v", name, prefixedProject(pid), err)
 	}
 	for retries := 0; retries < 3; retries++ {
-		ba, err = config.clientBilling.Projects.GetBillingInfo(prefixedProject(pid)).Do()
+		var ba *cloudbilling.ProjectBillingInfo
+		err = retryTimeDuration(func() (reqErr error) {
+			ba, reqErr = config.clientBilling.Projects.GetBillingInfo(prefixedProject(pid)).Do()
+			return reqErr
+		}, d.Timeout(schema.TimeoutRead))
 		if err != nil {
-			return err
+			return fmt.Errorf("Error getting billing info for project %q: %v", prefixedProject(pid), err)
```

Comment:
```suggestion
			return fmt.Errorf("error getting billing info for project %q: %v", prefixedProject(pid), err)
```

Reply:
I used the capital 'Error' to match the other strings in this file even though it is 'bad' go style. I thought consistency would be preferred?

Reply:
Yeah, either is fine.